### PR TITLE
[WIP] feature: build Misk using hermit

### PR DESCRIPTION
**Pull request by chris-ryan-square**
- [ ] Improve Hermit related docs - why, setup
- [ ] Change the Github actions to properly use Hermit commands

### CODE REVIEW

File: .github/workflows/Publish-Website.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
           java-version: 11.0.11+9
 
       - name: Generate API docs
-        run: ./gradlew dokkaGfm
+        run: gradle dokkaGfm
```
Comment (r3mariano) [inline .github/workflows/Publish-Website.yml:32]: How does this work (i.e. where is Hermit configured?)

File: build.gradle.kts [hunk 2]
```diff
@@ -10,16 +10,46 @@ buildscript {
...
+
+    }
+    classpath(Dependencies.wireGradlePlugin) {
+      exclude(group="org.apache.logging.log4", module="log4j-core")
+
+    }
+    classpath(Dependencies.kotlinBinaryCompatibilityPlugin) {
+      exclude(group="org.apache.logging.log4", module="log4j-core")
+
+    }
```
Comment (r3mariano) [inline build.gradle.kts:52]: What's this part about?

File: build.gradle.kts [hunk 3]
```diff
@@ -145,6 +175,8 @@ subprojects {
     if (name.contains("kapt") || name.contains("wire") || name.contains("proto") || name.contains("Proto")) {
       attributes.attribute(Usage.USAGE_ATTRIBUTE, this@subprojects.objects.named(Usage::class, Usage.JAVA_RUNTIME))
     }
+
+    exclude(group="org.apache.logging.log4", module="log4j-core")
```
Comment (r3mariano) [inline build.gradle.kts:179]: What's this part about?

File: docs/getting-started.md [hunk 4]
```diff
@@ -10,10 +10,12 @@ contains a Misk web app with the requisite dependencies.
 
 ## Start the service
 
+If you have not done so already, install and activate hermit: https://cashapp.github.io/hermit/
```
Comment (r3mariano) [inline docs/getting-started.md:13]: Probably want a little more description of activating the Hermit environment. Also wonder if directing people to use `bin/gradle` directly is good.

File: .github/workflows/Publish-Website.yml (hunk 1 above)
Comment (chris-ryan-square) [reply .github/workflows/Publish-Website.yml:32]: Good point.  I was partially trying this in the github builds to see what happens.

File: build.gradle.kts (hunk 2 above)
Comment (chris-ryan-square) [reply build.gradle.kts:52]: Hrm, I don't recall adding that

Comment (chris-ryan-square) [reply build.gradle.kts:52]: Oh wait - will fix.

Comment (chris-ryan-square) [reply build.gradle.kts:52]: This was an attempt to find what plugin was pulling in log4j.

File: docs/getting-started.md (hunk 4 above)
Comment (chris-ryan-square) [reply docs/getting-started.md:13]: You mean use show the commands as bin/gradle instead of just gradle?

**Pull request closed without merging**